### PR TITLE
FIX: Better handling for costs in access prompts

### LINKED ISSUE #3471: Runner forced to trash MVT even when on < 5 credits
Twice today in games the runner has been forced to trash MVT firing the CTM trace even when they had < 5 credits, and no other trash mechanism such as Imp/Freedome &c.

### PR DESCRIPTION
**Pull request by NoahTheDuke**
Fixes bug where Mumbad Virtual Tour would force trash when Runner had < 5 credits. Also fixes bug with not showing trash ability choices on agenda access prompts with multiple steal costs.

Fixes #3471, #2469

### CODE REVIEW

**Comment by Saintis** [comment]
For Imp + MVT, as the runner is force to trash MVT if installed and able I would expect prompts to give Pay and Imp option if >= 5 credits and just Imp otherwise (so no "No action" option). Is this possible with the current access stuff?

**Comment by NoahTheDuke** [comment]
I think you're right, let me fix that up.

File: src/clj/game/core/runs.clj [hunk 1]
```diff
@@ -234,39 +238,41 @@
...
+
+                                      ;; Steal normally
+                                      (= target "Steal")
+                                      (steal-agenda state :runner eid c)
+
+                                      ;; Pay single additiional cost to steal
+                                      (.contains target "Pay")
+                                      (if (> n 1)
+                                        ;; Use the better function for multiple costs
+                                        (resolve-ability state :runner
```
Comment (nealterrell) [inline src/clj/game/core/runs.clj]: I'm fairly certain this needs to be continue-ability. Otherwise this eid is left hanging.

Comment (NoahTheDuke) [reply src/clj/game/core/runs.clj]: Damn, good call. Thanks for catching that.

**Pull request merged**